go.sum

```diff
@@ -294,9 +301,33 @@ go.etcd.io/bbolt v1.3.5 h1:XAzx9gjCb0Rxj7EoqcClPD1d5ZBxZJk0jbuoPHenBt0=
 go.etcd.io/bbolt v1.3.5/go.mod h1:G5EMThwa9y8QZGBClrRx5EY+Yw9kAhnjy3bSjsnlVTQ=
 go.opencensus.io v0.21.0/go.mod h1:mSImk1erAIZhrmZN+AvHh14ztQfjbGwt4TtuofqLduU=
 go.opencensus.io v0.22.0/go.mod h1:+kGneAE2xo2IficOXnaByMWTGM9T73dGwxeWcUqIpI8=
+go.opentelemetry.io/contrib v0.20.0 h1:ubFQUn0VCZ0gPwIoJfBJVpeBlyRMxu8Mm/huKWYd9p0=
```

Comment:
I am not sure why all the imports are there, I tried running `go mod tidy` but did not help, but I guess internally opentelemetry relies on all of these?

Reply:
So... OpenTelemetry wants to make the modules related to tracing go to 1.0 without making the modules related to metrics go 1.0.  They decided to make smaller modules so to make that possible.  So I think the moldules below are correct.